superset/config.py

```diff
@@ -917,26 +919,29 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 MACHINE_AUTH_PROVIDER_CLASS = "superset.utils.machine_auth.MachineAuthProvider"
 
 # Email reports - minimum time resolution (in minutes) for the crontab
+#
+# Warning: This config key is deprecated and will be removed in version 2.0.0"
 EMAIL_REPORTS_CRON_RESOLUTION = 15
 
 # The MAX duration (in seconds) a email schedule can run for before being killed
 # by celery.
+#
+# Warning: This config key is deprecated and will be removed in version 2.0.0"
 EMAIL_ASYNC_TIME_LIMIT_SEC = 300
 
-# Email report configuration
-# From address in emails
-EMAIL_REPORT_FROM_ADDRESS = "reports@superset.org"
```

Comment:
Same comment with these config removals.

Reply:
This one is used anywhere, safe to just remove